### PR TITLE
Feature/remove blue bar

### PR DESCRIPTION
**Pull request by andrewsbusby**
# Description
[Explination](https://www.loom.com/share/a477f02e685740eb91767737b2218b1a)
Removing the blue bar in the middle of the page dividing the map section from the stats section of the page.
Fixes # (issue)
Removed stats folder and files. Removed import in Home.js to prevent any errors rendering the page.

## Type of change
A simple feature removal that wasn't necessary. 

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge

# Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -18,7 +17,6 @@ export default function Home() {
       <div className="bottom-section">
         {fetchStatus === 'success' ? (
           <>
```
Comment (pflee1989) [inline src/components/Home/Home.js:19]: Let's go ahead.

**Review by lncpadawan — COMMENTED** [review]
Thank you,  Andrew. Those files definitely needed to be erased, since we are no longer using the arrow and the blue bar at the bottom of the home page. Would have liked to see some pictures of what it looks like now :)

**Comment by pflee1989** [comment]
Great last step toward a MVP! Congrats for scoring this win for the team.

**Review by pflee1989 — APPROVED** [review]
Great change. Let's go ahead.

**Review by kcjerrell — APPROVED** [review]
Approved.

**Review by paulstgermain — CHANGES_REQUESTED** [review]
I'll need you to record the standard Loom walkthrough for this PR before I can commit to reviewing and merging it. Grab a free Loom account and step us through your work, explaining why you made this change, how you made this change, and using the files changed as a reference!

It doesn't have to be long - 5 minutes or less unless more time is required - but this is an important skill that will strengthen your skills as a developer and the amount of value you will bring to any team you join as well as making you a more capable interviewer!

**Comment by pflee1989** [comment]
https://www.loom.com/share/a477f02e685740eb91767737b2218b1a

**Review by paulstgermain — APPROVED** [review]

**Pull request merged**